### ISSUE TITLE
Posgresql deployment needs fixing

### ISSUE THREAD

**Issue by ivukotic** [ssl-hep/ServiceX#466]
Postgresql often uses > 1500 mcpus. 

In its pod description I find this:

 ```      
        app.kubernetes.io/name=postgresql
        helm.sh/chart=postgresql-11.6.26
  Service Account:  default
  Containers:
   postgresql:
    Image:      docker.io/bitnami/postgresql:14.4.0-debian-11-r23
    Port:       5432/TCP
    Host Port:  0/TCP
    Requests:
      cpu:      250m
      memory:   256Mi
```

There are two problems here: 
a) it needs to request 2 CPUs otherwise it will be kicked out once we have a lot of load from Transformers.
b) versions need to be fixed. We know we are deploying 14.4 but helm claims it is 11.6.26.

**Comment by sthapa** [comment]
For b, the helm chart version is 11.6.26.  However the underlying app version is 14.4.   There's a chart parameter that lets us change the version of postgresql that will be installed.  However, given that we've already released using 14.4, it'd be pretty disruptive to go back to 11.6.

**Comment by sthapa** [comment]
For a, that's the default setting from the postgresql subchart, a quick fix for this would be to update the defaults in the servicex chart.
